administrator-documentation/moderne-platform/how-to-guides/dev-center.md

```diff
@@ -0,0 +1,329 @@
+
+```graphql
+type Organization {
+    id: ID!
+    name: String!
+
+    """
+    Ordered list of commit options as they should appear in the UI.
+    """
+    commitOptions: [CommitOption!]!
+
+    parent: Organization
+
+    devCenter: DevCenter
+}
+```
+
+The [DevCenter object](TODO LINK WHEN READY) is the schema you need to follow in the below step to configure your DevCenter.
+
+## Step 3: Create and configure the DevCenter
+
+Your service must fulfill the [GraphQL contract mentioned in the previous step](https://github.com/moderneinc/moderne-organizations/blob/main/src/main/resources/schema/moderne-organizations.graphqls). If you chose to use [our template repository](https://github.com/moderneinc/moderne-organizations) for your Organizations service, all you will need to do is to create and configure your own [devcenter.json file](TODO ADD LINK WHEN READY).
+
+The `devcenter.json` file is where all of the configuration lies for DevCenters. In this file, you can configure things like which organizations should have a DevCenter, what cards should appear on said DevCenter, and what the keys should be on the cards. This file must follow the GraphQL schema mentioned above.
+
+{% hint style="success" %}
+When creating a DevCenter for the first time, we **strongly recommend** that you only create a DevCenter for a few key organizations. This will allow you to get data into the platform faster and ensure that you've configured everything correctly. Once everything is working as expected, you can then add more DevCenters as desired.
+{% endhint %}
+
+### Upgrade and migration cards
+
+Upgrade and migration cards allow you to see things like what Java version your repositories use, what version of Spring Boot they're on, or what JUnit version they use:
+
+![](/.gitbook/assets/migration-card.png)
+
+To create these cards, you will need three things:
+
+TODO: ADD LINKS TO THE BELOW BULLETS
+
+1. A relevant title for the card (e.g., `Spring Boot 3.2`)
+2. A recipe that can be run to fix the core issue the card is highlighting (e.g., `org.openrewrite.java.spring.boot3.UpgradeSpringBoot_3_2`)
+3. Some measures that break up the card into key categories and help your users determine a level of urgency. 
+
+#### Measures
+
+Each measure consists of a name and a recipe that can be used to determine whether or not a repository falls into said measure. This recipe _should not_ be the same recipe as the one used to "fix" the card. For instance, in a `Spring Boot 3.2` card, one measure might have a name of `Major` – which represents all repositories that are one or more major versions behind. The corresponding recipe would be `org.openrewrite.java.dependencies.DependencyInsight` and it would include the options of `1-2`. This would mark all repositories that use Spring Boot version `1.x` or `2.x` as being a major version or more behind.
```

Comment:
```suggestion
Each measure consists of a name and a recipe that can be used to determine whether or not a repository falls into said measure. We do not recommend that this recipe be the same recipe as the one used to "fix" the card. For instance, in a `Spring Boot 3.2` card, one measure might have a name of `Major` – which represents all repositories that are one or more major versions behind. The corresponding recipe would be `org.openrewrite.java.dependencies.DependencyInsight` and it would include the options of `1-2`. This would mark all repositories that use Spring Boot version `1.x` or `2.x` as being a major version or more behind.
```

I don't think it's necessary to tell them they *can't* use the same recipe as the fix recipe, but we recommend they use different measures so they can see the urgency of the changes required.